lib/parser.js

```diff
@@ -35,6 +35,41 @@ const blockOption = seq(
   }
 );
 const blockOptions = blockOption.many();
+
+//block
+const flagBegin = lexeme(string('#+BEGIN_'));
+const flagEnd= lexeme(string('#+END_'));
+const blockName = lexeme(regex(/[^\s]+/i));
+const content = lexeme(regex(/[^#+END_]*/));
+const block = seq(
+  startOfLine, flagBegin, blockName, singleLineText, content , flagEnd, singleLineText
+).map(
+  (val) => {
+    let ctt = val[4] ? val[4].split("\n") : 'None';
+    let type = "";
+    switch (val[2]) {
+      case "CENTER":
+        type = 'center';
+        break;
+      case "QUOTE":
+        type = 'quote';
+        break;
+      default:
+        type = 'special';
+        break;
+    }
+
+    ctt.pop();
+    return {
+      name: val[2],
+      type: type,
+      parameter: val[3]? val[3] : "None",
+      content: ctt,
+    };
+  }
+);
+const blocks = block.many();
```

Comment:
This assignment is unnecessary, use inline calling at https://github.com/weflex/node-orgmode/pull/9/files#diff-3fab227e34d65fe9bb2d6ea53eec41cfR327